README.md

```diff
@@ -1,2 +1,172 @@
-# Movies-App
-Here we create a movie app with two APIs
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="./MoviMAX.png" alt="logo" width="170"  height="auto" />
+  <br/>
+
+  <h3><b>MovieMax App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Javascript Capstone Project (Movie App) ](#-javascript-capstone-project-movie-app-)
+  - [👌 Kanban Template ](#-kanban-template-)
+  - [🎉 Zoom Video](#-zoom-video)
+  - [🛠 Built With HTML, CSS, \& Javascript](#-built-with-html-css--javascript)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Clone](#clone)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+# 📖 Javascript Capstone Project (Movie App) <a name="MovieMax"></a>
+
+> Js Capstone Project - This is a MovieMax app that sends and receives data from an API and populates data on the browser. It also allows you to add comments or likes to a movie. The app has implemented ES6 modules, callbacks, and promises, webpack, used AAA pattern for unit tests, and gitflow.
+
+## 👌 Kanban Template <a name="kanban">
+
+- Our Kanban template can be found [here.](https://github.com/felixDev22/Kanban---Movie-Project/projects/1)
+  
+## 🎉 Zoom Video
+- Our zoom video can be found [here.](https://drive.google.com/file/d/1E5l31oDKbgt5h0u7mVLxsu7HqJCZAkrv/view?usp=share_link)
```

Comment:
- [ ] Kindly repeat the video to highlight some pieces of code that you are proud of. 👇🏽 

> You should also highlight some interesting piece of code or something you built that you are very proud of.

Reply:
**Thank you very much for the suggested changes. These are the changes we made:**

- [x] We adjusted the nav bar to enable the movie counter to be on one line
- [x] Changed the display columns to 3
- [x] Added test for 0 counter
- [x] Added distance between genre and the movie poster
- [x] Added function to allow fast image load when the window is loaded
- [x] Repeat the video recording